.buildkite/pipeline.yml

```diff
@@ -38,23 +38,34 @@ steps:
               context: "Jetpack Prototype Build"
 
   #################
-  # Build the app
+  # Create Builds for Testing
   #################
-  - label: "🛠 Build for Testing"
-    key: "build"
-    command: ".buildkite/commands/build-for-testing.sh"
-    env: *common_env
-    plugins: *common_plugins
-    notify:
-      - github_commit_status:
-          context: "Build for Testing"
+  - group: "🛠 Builds for Testing"
+    steps:
+      - label: "🛠 :wordpress: Build for Testing"
+        key: "build_wordpress"
+        command: ".buildkite/commands/build-for-testing.sh wordpress"
+        env: *common_env
+        plugins: *common_plugins
+        notify:
+          - github_commit_status:
+              context: "WordPress Build for Testing"
+
+      - label: "🛠 :jetpack: Build for Testing"
+        key: "build_jetpack"
+        command: ".buildkite/commands/build-for-testing.sh jetpack"
+        env: *common_env
+        plugins: *common_plugins
+        notify:
+          - github_commit_status:
+              context: "Jetpack Build for Testing"
 
   #################
   # Run Unit Tests
   #################
-  - label: "🔬 Unit Tests"
-    command: ".buildkite/commands/run-unit-tests.sh"
-    depends_on: "build"
+  - label: "🔬 :wordpress: Unit Tests"
+    command: ".buildkite/commands/run-unit-tests.sh wordpress"
```

Comment:
Parameter not necessary anymore after the change in https://github.com/wordpress-mobile/WordPress-iOS/commit/5b78c495c4d8e052cf2220e4b6502dd981db943f, right?